admin_app/src/app/content/edit/page.tsx

```diff
@@ -224,6 +240,31 @@ const ContentBox = ({
             placeholder="Add Tags"
           />
         )}
+        filterOptions={(options, params) => {
+          const filtered = filter(options, params);
+
+          const { inputValue } = params;
+          const isExisting = options.some(
+            (option) => inputValue === option.tag_name
+          );
+          if (inputValue !== "" && !isExisting) {
+            filtered.push({ tag_id: 0, tag_name: `Add "${inputValue}"` });
```

Comment:
This is awesome! Can we somehow make sure the tag stays in original form and is not forced to be in caps?